docs/introduction/connectors.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: 'Connectors'
+slug: 'connectors'
+sidebar_position: 4
+---
+
+A connector is an external process which communicates with Conduit and knows how
```

Comment:
Alternatively, to avoid confusion due to the built-in connectors, maybe we can drop "external"?

Reply:
How about "_A connector runs in a separate process /.../_"?

Reply:
Is that still true for built-in connectors?

Reply:
Not really 🙈 I'll try to revise and clarify without overly complicating things. I'm thinking if it may even be better to just skip the "external process" altogether 🤔

Reply:
Yeah, maybe skipping the "external process" (how it works runtime) sounds good to me and the sentence is clear even without it.